src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -1481,7 +1481,7 @@ private function addHttpClientSection(ArrayNodeDefinition $rootNode)
                                         ->info('A comma separated list of hosts that do not require a proxy to be reached.')
                                     ->end()
                                     ->floatNode('timeout')
-                                        ->info('Defaults to "default_socket_timeout" ini parameter.')
+                                        ->info('The idle timeout, defaults to "default_socket_timeout" ini parameter.')
```

Comment:
Forgotten in https://github.com/symfony/symfony/pull/32928

Reply:
s/to/to the/